memdx/op_bootstrap.go

```diff
@@ -79,18 +83,23 @@ func (a OpBootstrap) Bootstrap(d Dispatcher, opts *BootstrapOptions, cb func(res
 		}
 	}
 
-	dispatchHello = func() {
+	pendingOp := &multiPendingOp{}
```

Comment:
I don't see anywhere that you remove ops from the multiPendingOp after they have completed.  I guess this is generally okay, since cancelling an already completed operation is a no-op.